blockstack-sdk/src/main/java/org/blockstack/android/sdk/ui/ConnectActivity.kt

```diff
@@ -0,0 +1,95 @@
+package org.blockstack.android.sdk.ui
+
+import android.content.Context
+import android.content.Intent
+import android.os.Bundle
+import androidx.annotation.StyleRes
+import androidx.appcompat.app.AppCompatActivity
+import androidx.lifecycle.coroutineScope
+import kotlinx.android.synthetic.main.activity_connect.*
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.ExperimentalCoroutinesApi
+import kotlinx.coroutines.FlowPreview
+import kotlinx.coroutines.launch
+import org.blockstack.android.sdk.BlockstackConnect
+import org.blockstack.android.sdk.R
+
+class ConnectActivity : AppCompatActivity() {
+
+    private val LAUNCH_HOW_IT_WORKS_ACTIVITY = 1
+
+    @FlowPreview
+    @ExperimentalCoroutinesApi
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_connect)
+
+        //Action Bar Close Icon & Title
+        supportActionBar?.setDisplayShowTitleEnabled(false)
+        supportActionBar?.setDisplayHomeAsUpEnabled(true)
+        supportActionBar?.setHomeAsUpIndicator(R.drawable.ic_close)
+
+        //Check if BlockstackConnect Config has a custom theme
+        setTheme(savedInstanceState?.getInt(
+                EXTRA_CUSTOM_THEME,
+                R.style.Theme_Blockstack) ?: R.style.Theme_Blockstack)
```

Comment:
You shouldn't need to do `?: R.style.Theme_Blockstack`, the default value is already provided.

Reply:
getInt is a optional.
`Type mismatch.
Required:
Int
Found:
Int?`